### PR TITLE
Seznam článků

### PR DESCRIPTION
**Pull request by JiriNenahlo**

### CODE REVIEW

**Comment by JiriNenahlo** [comment]
Prepared common article list component for all required roles, should speed up progress a lot after it's finished and merged.

**Review by legas1 — CHANGES_REQUESTED** [review]
WIP, do not merge

File: src/App.tsx [hunk 1]
```diff
@@ -47,6 +48,7 @@ function App() {
                         {navigation.rolePaths.map((rolePath, index) => {
                             return <PrivilegedRoute key={"privileged_route_" + index} exact path={rolePath.path} component={rolePath.component} role={rolePath.role} />;
                         })}
+                        <PrivilegedRoute key="show_article_id" path="/article/:id" role="ROLE_AUTHOR" component={ArticleDetail} />
```
Comment (legas1) [inline src/App.tsx]: remove the key plz

**Review by legas1 — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (JiriNenahlo) [reply src/App.tsx]: fucking react

**Review by legas1 — APPROVED** [review]

**Pull request merged**